pkg/ingress/endpoint/ep.go

```diff
@@ -60,5 +60,12 @@ func BuildEps(ns, name string, port int) []*v1.Node {
 			}
 		}
 	}
+	if len(nodes) < 1 {
```

Comment:
That's not a right behavior, if we really cannot collect the endpoint information, we shouldn't push this upstream to APISIX, instead, logging it and just throw it away.

Reply:
ok, we do synchronization before sync, if there is no node, we will try again.

Reply:
done